addons/point_of_sale/views/report_saledetails.xml

```diff
@@ -446,13 +446,13 @@
                     <br/>
                     <br/>
                     <t t-if="opening_note" id="opening_note">
-                        <strong>Opening of session note:</strong>
```

Comment:
restore the title. It's similar construction with "End of session note:". After you restore the title text, we can merge this PR.

Reply:
thanks @caburj ,
i was waiting for the test to be completed before pinging you.